### PR TITLE
[wip] New package: dart-sdk-2.4.0_1

### PR DESCRIPTION
**Pull request by Noah-Huppert**

### CODE REVIEW

**Comment by Hoshpak** [comment]
Packages should be built from source if the source is available, packages not built from source have to be suffixed with `-bin`. (and only be added if there's a good reason, e.g. bootstrapping) I don't think this package should be merged as is. Only supporting a single CPU architecture is another downside.

File: srcpkgs/dart-sdk/template [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+hostmakedepends="unzip"
+wrksrc=${pkgname}-${version}
+create_wrksrc=yes
+build_wrksrc=dart-sdk
+
+distfiles="https://storage.googleapis.com/dart-archive/channels/stable/release/${version}/sdk/dartsdk-linux-x64-release.zip"
+checksum="094c862f26ff2f203e5e95801281e9f60eeab3849aed1458f364ee757ed89287"
+
+do_install() {
+	     vlicense LICENSE
```
Comment (Johnnynator) [inline srcpkgs/dart-sdk/template]: Indentation should be one tab (you have 1 tab + 4 spaces here).

**Comment by Noah-Huppert** [comment]
I will update the pull request to use these build instructions: https://github.com/dart-lang/sdk/wiki/Building

**Comment by LinArcX** [comment]
@Noah-Huppert 
Any update?

**Comment by Anachron** [comment]
@Noah-Huppert please update the license and checksum.
Why  is this `x86_64`-only?

**Comment by Noah-Huppert** [comment]
No longer planning on completing this any time soon. I'll close this PR and make a new one if I do complete. Originally was only going to make it a bin package.

**Pull request closed without merging**